code/modules/research/techweb/all_nodes.dm

```diff
@@ -994,7 +994,7 @@
 	boost_item_paths = list()
 	for(var/path in GLOB.uplink_items)
 		var/datum/uplink_item/UI = new path
-		if(!UI.item || !UI.illegal_tech)
+		if(!UI.item || !UI.illegal_tech || ispath(UI.item, /obj/item/storage/box) || ispath(UI.item, /obj/item/storage/backpack))
```

Comment:
I'd rather you set illegal_tech = FALSE on all of these.

If you want it to not drift, consider a unit test.

Reply:
In another PR ( https://github.com/tgstation/tgstation/pull/58071 ), I am trying to get the techweb to look inside of storage objects.

If you buy the buzzkill grenades, the grenades should give illegal technology, not the box they come in. 

What would you recommend for a unit test?

Reply:
Forgot about this, sorry for the delay.

Basically, a unit test would just look through every uplink item, check if it is a box or a backpack, and if it is, ensure its illegal tech is off. This is preferable as it keeps the code here cleaner and without adding any additional (though in this case, menial) runtime cost.